client/package-lock.json

```diff
@@ -1,16069 +1,8 @@
 {
   "name": "client",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
You might be using an old version of npm as this first commit deletes most of the packagelock file. The expected version for this project is 14.15.1, which is shown in .nvmrc. If you don't have `nvm`, you can install it, and then `nvm use` to use the npm version that's in `.nvmrc.

Reply:
Yeah I do have nvm and I also used `nvm use`

Reply:
I tried this out locally. You're right, it's the nvm version in the file that was incorrect. I must've made a mistake here in using a newer NPM version from another project. Great catch!

I fixed this now in `main` so if you rebase against main and run `npm` install again, should be much less of a diff.